self-hosted/Dockerfile

```diff
@@ -22,8 +22,16 @@ RUN : \
 
 WORKDIR /usr/src/sentry
 
-ENV PATH=/.venv/bin:$PATH PIP_NO_CACHE_DIR=1 PIP_DISABLE_PIP_VERSION_CHECK=1
-RUN python3 -m venv /.venv
+ENV PIP_NO_CACHE_DIR=1 PIP_DISABLE_PIP_VERSION_CHECK=1 \
+    UV_COMPILE_BYTECODE=1 UV_NO_CACHE=1
+
+ENV PATH="/usr/src/sentry/.venv/bin:$PATH"
```

Comment:
I thought you weren't going to change the virtualenv location?

Reply:
I wasn't, thanks for catching :P swear i had a commit for this somewhere